2014-09-05 11:47:53  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:156891 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 11:47:54  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:157766 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 11:50:19  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:302005 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 11:50:23  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:306632 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:21:55  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9398574 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:21:55  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9398574 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:21:56  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:9399915 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:21:58  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:9401586 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:21:58  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:9401618 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:21:59  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9402569 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:21:59  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9402867 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:01  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:9404286 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:01  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:9404350 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:02  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:9405239 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:02  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9405255 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:02  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9405941 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:02  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:9405941 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:03  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:9406940 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:04  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:9407003 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:04  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9407924 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:05  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9408222 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:07  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:9410281 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:07  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:9410422 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:08  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:9411483 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:08  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9411499 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:09  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9412451 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:09  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:9412576 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:10  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:9413638 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:10  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:9413732 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:11  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9414575 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:11  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9414575 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:12  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:9415512 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:12  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:9415512 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:13  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:9416619 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:13  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9416683 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:15  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9418275 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:15  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:9418275 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:15  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:9418540 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:15  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:9418540 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:16  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9419554 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:16  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9419570 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:17  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:9420741 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:17  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:9420756 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:18  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:9421865 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:18  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9421865 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:19  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9422630 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:20  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:9423145 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:21  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:9424238 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:21  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:9424254 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:22  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9425347 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:22  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9425347 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:22:23  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:9426206 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:23:05  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:9468295 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:23:06  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:9469294 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:24:53  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:9576392 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:24:54  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:9577449 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:25:53  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:1897 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:25:54  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:2881 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:55:20  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:3990 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:56:59  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:102959 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:57:00  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:103955 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 14:59:04  [ org.huangxu.service.Server.startService(Server.java:39):org.huangxu.service.Server:main:0 ] - [ ERROR ]  Could not start services. Status is: false
java.lang.ArrayIndexOutOfBoundsException: 12288
	at org.huangxu.service.DbService.init(DbService.java:61)
	at org.huangxu.service.Server.startService(Server.java:30)
	at org.huangxu.service.Server.init(Server.java:90)
	at org.huangxu.Booter.main(Booter.java:23)
2014-09-05 14:59:04  [ org.huangxu.service.Server.init(Server.java:91):org.huangxu.service.Server:main:0 ] - [ ERROR ]  Can't start service !
2014-09-05 15:00:07  [ org.huangxu.service.Server.startService(Server.java:39):org.huangxu.service.Server:main:1 ] - [ ERROR ]  Could not start services. Status is: false
java.lang.ArrayIndexOutOfBoundsException: 12288
	at org.huangxu.service.DbService.init(DbService.java:61)
	at org.huangxu.service.Server.startService(Server.java:30)
	at org.huangxu.service.Server.init(Server.java:90)
	at org.huangxu.Booter.main(Booter.java:23)
2014-09-05 15:00:07  [ org.huangxu.service.Server.init(Server.java:91):org.huangxu.service.Server:main:5 ] - [ ERROR ]  Can't start service !
2014-09-05 15:01:57  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:42846 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:01:58  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:43650 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:02:01  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:47130 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:02:02  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:48143 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:04:27  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:192557 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:04:28  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:193587 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:11:09  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:67747 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:11:10  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:68719 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:11:44  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:103040 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:11:45  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:104024 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:12:42  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:160719 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:12:43  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:161766 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:12:51  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:169881 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:12:52  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:170909 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:14:12  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:251147 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:14:14  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:252835 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:33:15  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:16849 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-05 15:33:16  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:17817 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
